play.pokemonshowdown.com/src/panel-chat.tsx

```diff
@@ -913,10 +913,21 @@ export class ChatTextEntry extends preact.Component<{
 				PS.leave(PS.room.id);
 				return true;
 			}
-		// } else if (e.keyCode === 32 && PS.user.lastPM && ['/reply', '/r', '/R'].includes(this.getValue())) { // '/reply ' is being written
-		// 	const newValue = `/pm ${PS.user.lastPM}, `;
-		// 	this.setValue(newValue, newValue.length);
-		// 	return true;
+			// } else if (e.keyCode === 32 && PS.user.lastPM && ['/reply', '/r', '/R'].includes(this.getValue())) { // '/reply ' is being written
+			// 	const newValue = `/pm ${PS.user.lastPM}, `;
+			// 	this.setValue(newValue, newValue.length);
+			// 	return true;
+		} else if (ev.shiftKey && ev.keyCode === 37) {
+			const toIndex = PS.rightRoomList.indexOf(PS.room.id) - 1;
+			PS.moveRoom(PS.room, 'right', false, toIndex);
```

Comment:
Would this work if the room wasn't a right room? Plausibly the buttons should be different in vertical tabs mode.